validations.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# require_relative("event")
+require('time')
+require('date')
+require 'io/console'
+# Validations for validations and helper methods
+module Validations
+  
+
+  def validate_date(date)
+    return nil if (date =~ %r(^\d{2}/\d{2}/\d{4})).nil? && (date =~ %r(^\d{2}/\d{4})).nil?
+
+    Date.parse(date)
```

Comment:
use Date.parse(date) rescue nil

Reply:
I think there is no need to rescue nil at Date.parse(date) because the regex checks the date format. If the date isnt in dd/mm/yyyy format it returns nil there and doesnt come to that